articles/services/premier-success.md

```diff
@@ -0,0 +1,86 @@
+
+The Kickoff/Onboarding session is performed onsite with your dedicated Auth0 Premier Success Team which includes a CSM, a CSA, and an architecture engineer (AE). The goal of this session is to work with you to agree on a plan of working with Auth0. The session will include the following topics:
+
+* Review of your success resources and processes including systems, contacts, documentation, and reporting.
+* Confirm your projects and goals
+* Review your ROI and success criteria
+* Begin your success plan and interaction cadence (Touchpoints/QBR)
+
+## Solutions Workshop
+
+The Solutions Workshop is a monthly, three hour, CSA-led workshop via Zoom for the first six months of the term of the subscription, and quarterly thereafter. The goal of this session is for the Auth0 architect and key technical resources to do the following:
+
+* Current project status
+* Technology/product challenges
+* Identity architecture
+* Solution design and best practices
+* Future project goals, architecture, and solutions
+* Feature guidance and awareness
+
+## Business Review
+
+The Business Review is a quarterly onsite review session with your Auth0 Premier Success Team (CSM, CSA, and AE) that includes a tailored product roadmap review. Your Auth0 Premier Success Team and selected advisors conduct a thorough business review for your company. This review is strategic in nature, not tactical! Auth0 suggests a quarterly cadence as an industry standard, tailored to the meet your needs. This review will cover the following topics:
+
+* ROI using Auth0
+* Expansion of the value proposition from client perspective
+* Additional use cases you might be exploring
+* Product roadmap
+* Support, product usage and NPS scorecards
+
+## Engagement Model
+
+The Engagement Model provides invitations to live webinars, personalized office hours, tailored customer outreach, and access to the Auth0 beta program. The goal of the Engagement Model is to build a relationship with you to help you reach your business goals. You will be invited to the following Auth0 programs:
+
+* Webinars
+* Office hours on best practices
+* Consistent customer outreach
+* Early access program eligibility
+
+## Success Plan
+
+Your Premier Success Team will develop, track, and deliver a strategic Success Plan to keep you on track to achieve your ROI. The key topics of the plan include:
+
+* Roll-up of all your details about use cases, subscription information, legacy type, contractual obligations, targeted go-live dates and license roll-out plans.
+* Program dates and project timelines
+* Identified outcomes and measurable goals
+* Knowledge transfer and learning opportunities 
+
+## Operations and Dashboard Review
+
+An Operations and Dashboard Review is provided monthly. The goal of this review is to better understand and align with you around issues, upcoming activities, Auth0 change management, and issue trending/analysis. Key topics include:
+
+* Support ticket review and analysis
```

Comment:
Lets remove this line on ticket reviews. In speaking with Dave Wilner he didn't want us to highlight ticket review

Reply:
Done